src/utils/daemon.rs

```diff
@@ -220,38 +223,44 @@ pub fn start_daemon(background: bool, config: Arc<Config>) -> Result<(), Error>
     }).unwrap();
 
     // update release activity everyday at 23:55
-    thread::Builder::new()
-        .name("release activity updater".to_string())
-        .spawn(move || loop {
-            thread::sleep(Duration::from_secs(60));
-            let now = Utc::now();
-
-            if now.hour() == 23 && now.minute() == 55 {
-                info!("Updating release activity");
-                if let Err(e) = update_release_activity() {
-                    error!("Failed to update release activity: {}", e);
-                }
-            }
-        })
-        .unwrap();
+    let cloned_db = db.clone();
+    cron("release activity updater", 60, move || {
```

Comment:
Let's avoid having an implicit 'seconds' here, can you change `cron` to take `Duration` instead of `u64`?